contracts/types/WeightedSigners.sol

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+/**
+ * @notice This struct represents the weighted signers payload
```

Comment:
Separating the type definitions. We'll look into exposing Rust bindings automatically for these to make it more easy for Amplifier to be in sync